CHANGELOG_PENDING.md

```diff
@@ -19,3 +19,6 @@
 - Allow Fn::ToBase64 to take expressions as inputs, was previously constrained to only allow a
   string constant.
   [#221](https://github.com/pulumi/pulumi-yaml/pull/221)
+
+- [expr] Fix handling of "plain" input maps when sending properties to component providers such as AWSX.
+  https://github.com/pulumi/pulumi-yaml/pull/195
```

Comment:
We seem to have a format for the changelog. I would prefer we remain consistent:
```suggestion
  [#195](https://github.com/pulumi/pulumi-yaml/pull/195)
```